CONTRIBUTING.md

```diff
@@ -120,3 +122,21 @@ If you add a migration to the DB, test it with this gradle task
 ```shell
 ./gradlew app:connectAndroidTest
 ```
+
+## Updating the Baseline profiles
+
+You can generate the latest baseline profiles by executing the following gradle task.
+It will generate the baseline profiles and the startup profiles.
```

Comment:
Should be on the same line, unless it's a new paragraph.

Reply:
The advantages of Markdown is that you can write your sentences line by line but they will be properly merged in the rendered document.

![image](https://github.com/LemmyNet/jerboa/assets/67873169/e4baa08d-dfe0-4491-89f1-aed810a98130)

Reply:
I've always found that weird bc it makes viewing the file in an editor very different from how it loovs rendered. Up to you I spose.